test/unit/test-optimize-batch.js

```diff
@@ -35,3 +36,10 @@ test('Sum of optimized batches equals input', t => {
 
   t.is(_.sum(_.flatten(o)), 66)
 })
+
+test('Should return input array', t => {
+  const input = [7,15]
+  const o = optimizeDispenseWithoutLimit(input)
+  
+  t.deepEqual(o, input)
+})
```

Comment:
This test will fail after https://github.com/lamassu/lamassu-machine/pull/195 gets merged. It should return [[7,15]].